src/components/VmDetails/cards/DisksCard/DiskStateIcon.js

```diff
@@ -12,19 +12,19 @@ const diskStateSettings = {
     type: 'fa',
     name: 'arrow-circle-o-up',
     className: style['state-icon-active'],
-    tooltip: msg.diskStateActiveTooltip(),
+    tooltip: `${msg.status()}: ${msg.diskStateActiveTooltip()}`,
```

Comment:
Nit-pick:  Not all languages will naturally go "A: B" so it is better form (but not required) to embed the tooltip value in the status string:

```js
  `${msg.status({ statusInfo: msg.diskStateActiveTooltip() })}`
```

Then the string in messages.js would look like:
```
  status: "Status: {statusInfo}"
```

Reply:
Thanks :)